components/placeholder/test/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Placeholder from '../';
```

Comment:
That is the problem it is Dashicon.  I am switching these tests over to use the proper `{ Placeholder } from 'components'` which does work.  There is a dependency on wp.element as an external in Dashicon.